### PR TITLE
This is regrading to PRE-797.

### PR DESCRIPTION
**Pull request by shreyascelestial**
Made changes to enable the hook-check and hook-response functionality and added a couple of scripts for testing as a reference for future use.
I have made changes in the build.gradle file to disable obfuscation. When obfuscation is enabled, it becomes difficult to inject scripts into the appropriate classes or methods due to the obfuscated code note has been added to Readme file to turn of the obfuscation.
please do review 
jira link: https://sembi.atlassian.net/browse/PRE-797

### CODE REVIEW

**Comment by branflake2267** [comment]
lgtm. Nice job!

File: android/HookingOnAndroid/build.gradle [hunk 1]
```diff
@@ -62,11 +62,11 @@ android {
     buildTypes {
         release {
             signingConfig signingConfigs.release
-            minifyEnabled true
+            minifyEnabled false
```
Comment (hectorlugosembi) [inline android/HookingOnAndroid/build.gradle]: I believe that we need to have this as true . If the value is false we are not applying the whole security.

File: android/HookingOnAndroid/build.gradle [hunk 2]
```diff
@@ -62,11 +62,11 @@ android {
...
         }
 
         debug {
-            minifyEnabled true
+            minifyEnabled false
```
Comment (hectorlugosembi) [inline android/HookingOnAndroid/build.gradle]: I believe that we need to have this as true . If the value is false we are not applying the whole security.

File: android/HookingOnAndroid/build.gradle (hunk 1 above)
Comment (shreyascelestial) [reply android/HookingOnAndroid/build.gradle]: done, i have exclude the those class to perform the hook check and hook response.

File: android/HookingOnAndroid/build.gradle (hunk 2 above)
Comment (shreyascelestial) [reply android/HookingOnAndroid/build.gradle]: done, i have exclude the those class to perform the hook check and hook response.

File: android/HookingOnAndroid/README.md [hunk 3]
```diff
@@ -21,6 +21,8 @@ Feel free to reconfigure the probability and/or responses of the `HookResponse`
 
 This sample includes a keystore, `keystore.ks`, that is used to sign the release build.
 
+#### Note: To ensure that the `HookCheck` and `HookResponse` features work correctly, please disable obfuscation in the build.gradle file
```
Comment (hectorlugosembi) [inline android/HookingOnAndroid/README.md:24]: is this note needed ? now that we have  "minifyEnabled true" If this is not needed remove it.

**Review by hectorlugosembi — APPROVED** [review]
PR approved I added a comment that is not a blocker. But please address it and let me know.

File: android/HookingOnAndroid/README.md (hunk 3 above)
Comment (shreyascelestial) [reply android/HookingOnAndroid/README.md:24]: Removed.

**Comment by shreyascelestial** [comment]
Thanks @hectorlugosembi, @branflake2267  i will merge this PR.

**Pull request merged**